web/js/font-awesome-library.js

```diff
@@ -0,0 +1,134 @@
+/*
+   Provides a single import location for font-awesome icons.  This allows us
+   to simply import the FontAwesomeIcon component where we want to display
+   an icon and only pass the string name of an icon.
+
+   For example:
+      import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+      <FontAwesomeIcon icon="times" fixedWidth />
```

Comment:
Just out of curiosity... Is this a more performant solution, or more of an enhancement in terms of future development?

Reply:
I don't think there is a difference in performance but I think only having to ever import an icon once seemed tidier.  That way, for any component that needs to show an icon, you only need to ever import `<FontAwesomeIcon>`